src/contexts/TodoContext.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo, useState } from 'react';
+import { Todos } from '../types/Todos';
+import { Status } from '../types/Status';
+import { filterTodoByStatus } from '../services/FilteringByStatus';
+import { ContextProps } from '../types/ContextProps';
+import { useLocalStorage } from '../hooks/LocalStorage';
+
+export const TodoContext = React.createContext<ContextProps>({
+  todos: [],
+  addTodo: () => { },
+  deleteTodo: () => { },
+  setTodos: () => { },
+  filterTodoByStatus: () => [],
+  setStatus: () => [],
+  status: Status.All,
+  clearTodo: () => [],
+  editTodo: () => [],
+  toggleAll: () => {},
+});
+
+type Childer = {
```

Comment:
please fix mistake in this word `Childer`) 
 
I would recommend to write word Props in the end like `FormikAddressProps`, `AdvisorsTableProps`